spec/unit/templates/10-clusterDefaults_spec.rb

```diff
@@ -0,0 +1,150 @@
+        } 
+      }
+      it 'sets a relative disk alarm' do
+        expect(rendered_template).to include('disk_free_limit.relative = 1.5')
+        expect(rendered_template).not_to include('disk_free_limit.absolute')
+      end
+    end
+
+    context 'when provided an absolute threshold' do
+      context 'when the threshold is just a number' do
+        let(:manifest_properties) { {
+            'rabbitmq-server' => {
+              'disk_alarm_threshold': '1000000000'
+            }
+          } 
+        }
+        it 'sets an absolute disk alarm' do
+          expect(rendered_template).to include('disk_free_limit.absolute = 1000000000')
+          expect(rendered_template).not_to include('disk_free_limit.relative')
+        end
+      end
+      context 'when the threshold is with units included' do
+        let(:manifest_properties) { {
+            'rabbitmq-server' => {
+              'disk_alarm_threshold': '23GB'
+            }
+          } 
+        }
+        it 'sets an absolute disk alarm' do
+          expect(rendered_template).to include('disk_free_limit.absolute = 23GB')
+          expect(rendered_template).not_to include('disk_free_limit.relative')
+        end
+      end
+    end
+  end
+
+  context 'when a cluster name is provided' do
+		let(:manifest_properties) { {
+				'rabbitmq-server' => {
+          'cluster_name': 'my-favourite-rabbit'
+        }
+      } 
+    }
+    it 'configures the cluster name' do
+      expect(rendered_template).to include('cluster_name = "my-favourite-rabbit"')
+    end
+  end
+
+  context 'when definitions are provided to the rabbitmq-server job' do
+		let(:manifest_properties) { {
+				'rabbitmq-server' => {
+          'load_definitions': '{"hello":"world","my_name_is":73}'
+        }
+      } 
+    }
+    it 'configures RabbitMQ to import the definitions' do
+      expect(rendered_template).to include('load_definitions = /var/vcap/jobs/rabbitmq-server/etc/definitions.json')
+
+    end
+  end
+
+  context 'when the BOSH release is set to perform hot swap' do
+  let(:rendered_template) do
+    links = {
+      'rabbitmq-server' => {
+        'instances' => [
+          { "address" => 'rabbit1.foo.com' },
+          { "address" => 'rabbit2.foo.com' }
+        ],
+        'properties' => {
+          'rabbitmq-server' => {
+            'ports' => [ 123, 456, 789, 10000000 ],
+            'timeouts' => [[123, "30s"]]
+          }
+        }
+      }
+    }
+
+    compiled_template('rabbitmq-server', 'config-files/10-clusterDefaults.conf', manifest_properties, links)
+  end
+		let(:manifest_properties) { {
+				'rabbitmq-server' => {
+          'create_swap_delete': true
+        }
+      } 
+    }
+    it 'configures RabbitMQ to import the definitions' do
+      expect(rendered_template).to include('cluster_formation.classic_config.nodes.1 = rabbit@rabbit1.foo.com')
+      expect(rendered_template).to include('cluster_formation.classic_config.nodes.2 = rabbit@rabbit2.foo.com')
+    end
```

Comment:
This test is about cluster formation, the `it` description doesn't seem accurate. Or a copy-paste overlook? 😄